src/List.js

```diff
@@ -13,7 +13,28 @@ import { Link } from 'react-router-dom';
 import { NavigationMenu } from './NavigationMenu';
 import { useHistory } from 'react-router-dom';
 
-const convertToDays = (num) => num / 1000 / 60 / 60 / 24;
+const convertToDays = (num) => Math.round(num / 1000 / 60 / 60 / 24);
+
+const daysSinceLastPurchaseOrCreationTime = (item) =>
+  convertToDays(new Date() - (item.lastPurchasedDate || item.creationTime));
+
+const itemIsInactive = (item) =>
+  daysSinceLastPurchaseOrCreationTime(item) > 2 * item.previousEstimate ||
+  item.totalPurchases === 1;
+
+const itemState = (item) => {
```

Comment:
`State` is a pretty broad term. Should we make this more specific in its naming to something like `getClassName`?

Reply:
Thank you!!

Reply:
I'm still seeing it called `itemState`. I'm good with whatever and am fine if you don't want to change it. Just an idea...

Reply:
Hrm... we changed it 2 commits ago...wonder if this has something to do with the accidental merge into main that you're seeing the older one...

Here is the commit in case interested:
https://github.com/the-collab-lab/tcl-31-smart-shopping-list/commit/4876a0de91ce400e93e43aa00b42108b87900937

(ps. loving the "atomic / micro commits" we talked about last week, really useful for finding things!)